### PR TITLE
Add connector filter

### PR DESCRIPTION
**Pull request by mahdibenromdhane**

### CODE REVIEW

File: src/app/shared/model/charging-stations.model.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import { KeyValue } from 'types/GlobalType';
+import { Utils } from 'utils/Utils';
+
+export const CONNECTORS: KeyValue[] = [
```
Comment (LucasBrazi06) [inline src/app/shared/model/charging-stations.model.ts:4]: You can automate this with an new Array(26)...

Comment (mahdibenromdhane) [reply src/app/shared/model/charging-stations.model.ts]: Done.

File: src/app/pages/charging-stations/list/charging-stations-list-table-data-source.ts [hunk 2]
```diff
@@ -329,6 +330,7 @@ export class ChargingStationsListTableDataSource extends TableDataSource<Chargin
         new IssuerFilter().getFilterDef(),
         new SiteTableFilter().getFilterDef(),
         new SiteAreaTableFilter().getFilterDef(),
+        new ConnectorTableFilter().getFilterDef()
```
Comment (LucasBrazi06) [inline src/app/pages/charging-stations/list/charging-stations-list-table-data-source.ts:333]: Remove from charging station list and add it in Session lists as it is mentioned in the issue name  ;-)

**Review by LucasBrazi06 — CHANGES_REQUESTED** [review]

File: src/app/pages/charging-stations/list/charging-stations-list-table-data-source.ts (hunk 2 above)
Comment (mahdibenromdhane) [reply src/app/pages/charging-stations/list/charging-stations-list-table-data-source.ts]: My fault ! It's good now.

**Review by LucasBrazi06 — APPROVED** [review]

**Pull request merged**